runtime/src/main/java/org/corfudb/runtime/collections/TxnContext.java

```diff
@@ -451,6 +452,32 @@ K enqueue(@Nonnull Table<K, V, M> table,
         return ret;
     }
 
+    /**
+     * This API is used for LR Routing Queue model.
+     *
+     * This API is needed because the queue's id captures the sequencer's address which
```

Comment:
This description can be changed to something like - 'This API is used to add an entry to the CorfuQueue without materializing the queue in memory'

Other description explains the reason for using a queue and is not required as a javadaoc

Reply:
Changed!